jsrelay/.env

```diff
@@ -6,9 +6,10 @@ RELAY_HUB=0x79D1160DFDc24326a21fDC280D8FEc341c85D9Af
 #can't use localhost: must specify device IP
 #NODE_URL=http://192.168.1.241:8545
 
-GAS_PRICE_PERCENT=70
-BASE_FEE=
-PERCENT_FEE=
+#GAS_PRICE_PERCENT=70
+SERVER_CONFIG_FILE=server-config.json
+#BASE_FEE=
+#PERCENT_FEE=
```

Comment:
I talked with @drortirosh about that, and he believes that having multiple ways to configure it might be beneficial for using other systems than docker/docker-compose, and I don't mind it either now.